app/controllers/views/peeps/peeps.erb

```diff
@@ -0,0 +1,16 @@
+<ul>
+<a href ='/peeps/new'>Peep</a>
+</ul>
+
+
+
+<ul>
+  <% @peeps.reverse_each do |peep| %>
```

Comment:
Love the reversal here! I did it at the SQL level, but I think this makes it much clearer on the View level that it's in reverse chronological order.